chevah/empirical
# [#3470] Add support for SLES 10.

Scope
=====

This add the sles 10 builder on empirical.

Changes
=======

the build system was updated as sles 10 has an old sudo


How to try and test the changes
===============================

reviewers: @alibotean 

check that changes make sense
